sonic_platform_base/sonic_xcvr/api/public/cmis.py

```diff
@@ -460,7 +458,7 @@ def get_tx_bias(self):
         if tx_bias_support is None:
             return None
         if not tx_bias_support:
-            return ["N/A" for _ in range(self.NUM_CHANNELS)]
+            return {'LaserBiasTx%dField' % i : "N/A" for i in range(1, self.NUM_CHANNELS+1)}
```

Comment:
This will fail sonic-mgmt test since the API currently expects a list.

Reply:
thanks. now fixed.